native/cocos/audio/android/PcmAudioService.cpp

```diff
@@ -27,6 +27,7 @@ THE SOFTWARE.
 
 #include "audio/android/PcmAudioService.h"
 #include "audio/android/AudioMixerController.h"
+#include "audio/android/utils/Compat.h"
```

Comment:
Why do we need this?

Reply:
Part of the code compatible with openharmony and android will be defined in it